pyproject.toml

```diff
@@ -15,3 +15,8 @@ markers = [
 ]
 testpaths = ["tests"]
 norecursedirs = [ ".*", "build", "dist", "*.egg", "data", "__pycache__", ".github", "nf_core", "docs"]
+
+[tool.isort]
+profile = "black"
+known_first_party = ["nf_core"]
+force_single_line = true
```

Comment:
```suggestion
```

https://nfcore.slack.com/archives/CE5LG7WMB/p1653030965720379

<img width="255" alt="image" src="https://user-images.githubusercontent.com/465550/169496730-b45a1668-23c6-4a2d-a249-79e6af0b97e0.png">

![image](https://user-images.githubusercontent.com/465550/169496994-834a7aaa-e5df-4a93-ad31-321588c46d0e.png)

Reply:
But replace it with what the council has approved.
```suggestion
multi_line_output = 3
```